PlayerUI/Views/PUIPlayerView.swift

```diff
@@ -1140,15 +1140,37 @@ public final class PUIPlayerView: NSView {
 
     private var keyDownEventMonitor: Any?
 
-    private enum KeyCommands: UInt16 {
-        case spaceBar = 49
-        case leftArrow = 123
-        case rightArrow = 124
-        case minus = 27
-        case plus = 24
-        case j = 38
-        case k = 40
-        case l = 37
+    private enum KeyCommands {
+        case spaceBar
+        case leftArrow
+        case rightArrow
+        case minus
+        case plus
+        case j
+        case k
+        case l
+
+        static func fromEvent(_ event: NSEvent) -> KeyCommands? {
+            switch event.keyCode {
+            case 123: return .leftArrow
+            case 124: return .rightArrow
+            default: break
+            }
+            
+            guard let character = event.charactersIgnoringModifiers else {
```

Comment:
Reading through the docs, `keyCode` and `charactersIgnoringModifiers` both say they they can raise exceptions if the event is not a key event.

`addLocalMonitorForEvents(matching: .keyDown)` should ensure it's always a key event, but this function could use a `guard` for the type of event for thoroughness.

To increase the accuracy of this function, we could also ensure no modifiers are pressed as well via `event.modifierFlags.isEmpty`.